imap_processing/mag/mag_cdf_attrs.py

```diff
@@ -42,6 +42,38 @@
     instrument_base=mag_base,
 )
 
+mag_l1a_norm_mago_attrs = GlobalDataLevelAttrs(
+    "L1A-raw-norm-mago>Level 1A MAGo normal rate",
+    # Should also include data type
+    logical_source="imap_mag_l1a_norm-mago",
+    logical_source_desc="IMAP Mission MAG Instrument Level-1A Data.",
```

Comment:
If these are all going to be the same, you could make a `mag_l1a_desc` variable for this string to avoid some redundancy? If you think it improves readability. i.e.

```python
mag_l1a_desc = "IMAP Mission MAG Instrument Level-1A Data."
logical_source_desc=mag_l1a_desc
```

Reply:
I actually suspect that this should specify norm/burst/mago/magi in it... 🤔

Reply:
I think they should probably all be a little different, like this one should be IMAP Mission MAG O (whatever "O" stands for) Normal Rate Level-1A Data

Reply:
Oh we made this comment just a few minutes apart lol, I hadn't seen that you wrote when I made my comment.

Reply:
O stands for outboard, I stands for inboard